### PR TITLE
{bio}[GCC/10.3.0] Arriba v2.1.0

### PR DESCRIPTION
**Pull request by SethosII**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by robert-mijakovic** [comment]
@boegelbot please test @ generoso

**Comment by robert-mijakovic** [comment]
Test report by @robert-mijakovic
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
login01.meluxina.lxp.lu - Linux RHEL 8.3, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/7f797565dc868bad9a969ea8ede7aa97 for a full test report.

File: easybuild/easyconfigs/a/Arriba/Arriba-2.1.0-GCC-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+dependencies = [
+    ('bzip2', '1.0.8'),
+    ('STAR', '2.7.9a'),
+    ('zlib', '1.2.11'),
+    ('XZ', '5.2.5'),
+]
+
+postinstallcmds = ['mkdir %(installdir)s/database']
+files_to_copy = ['%(namelower)s', 'documentation', 'download_references.sh', 'draw_fusions.R', 'LICENSE',
```
Comment (verdurin) [inline easybuild/easyconfigs/a/Arriba/Arriba-2.1.0-GCC-10.3.0.eb:29]: @SethosII bit of a nitpick, but could you make this first line shorter please?

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/a/Arriba/Arriba-2.1.0-GCC-10.3.0.eb (hunk 1 above)
Comment (SethosII) [reply easybuild/easyconfigs/a/Arriba/Arriba-2.1.0-GCC-10.3.0.eb]: @verdurin No problem, I changed it.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.5
See https://gist.github.com/cd1e33b4ab2df9a421b2628b56dccbb4 for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @SethosII!

**Pull request merged**